using-timescaledb/continuous-aggregates.md

```diff
@@ -151,6 +151,13 @@ will be spawned immediately following completion. This means that the part of th
 data processed by previous jobs will be available even before the
 materialization is fully caught up.
 
+The `timescaledb.ignore_invalidation_older_than` parameter control how
```

Comment:
Should we add a new subsection here:  "More ways to control continuous aggregate?" or something similar. Then add ignore_invalidation_older_than and the parameter from Sven's JIT aggregates in this scetion.

Reply:
We probably need an additional section for JIT, yes, but I should add that as a separate PR.